src/vue/backend/src/App.vue

```diff
@@ -51,12 +53,17 @@ export default {
         ])
             .then((results) => {
                 requiredProducts.forEach((product, index) => {
-                    this.handleResults(results[index], product)
+                    if (product.module !== 'omnibar' && product.module !== 'importContent') {
+                        this.handleResults(results[index], product)
+                    }
                 })
             })
             .catch((error) => {
-                /* Todo: handle error - pass to state? */
                 console.log('error %o', error)
+                this.$store.dispatch('alerts/setItem', { key: 'notification', value: { show: true, error: true, text: 'We\'re experiencing a problem, please try again' } })
+                setTimeout(() => {
+                    this.$store.dispatch('alerts/setItem', { key: 'notification', value: { show: false, error: true, text: 'We\'re experiencing a problem, please try again' } })
```

Comment:
So when you catch the error, you dispatch an event to commit state to set and display an error notification to please try again. Then 3 seconds later hide it, but set the same values?

Reply:
Yeah, resetting the same values keeps it from doing a weird visual effect where the header collapses then disappears.

Reply:
👍